kubernetes/zenko/charts/redis-ha/values.yaml

```diff
@@ -59,30 +63,17 @@ securityContext:
 ## Node labels, affinity, and tolerations for pod assignment
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
 ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
-nodeSelector: {}
-podAnnotations: {}
-extraLabels: {}
-tolerations: []
-affinity: |
-  podAntiAffinity:
-    requiredDuringSchedulingIgnoredDuringExecution:
-    - labelSelector:
-        matchLabels:
-          app: {{ template "redis-ha.name" . }}
-          release: {{ .Release.Name | quote }}
-      topologyKey: kubernetes.io/hostname
+## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+podAntiAffinity: soft
 
 podDisruptionBudget:
   maxUnavailable: 1
 
-## Redis password - Not implemented
-## Defaults to a random 10-character alphanumeric string if not set and auth is true
-## ref: https://github.com/kubernetes/charts/blob/master/stable/redis-ha/templates/redis-auth-secret.yaml
-##
-## redisPassword:
+# redisPassword:
+auth: false
 
 persistentVolume:
-  enabled: true
+  enabled: false
```

Comment:
As per the review guidelines https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md
"Persistence should be enabled by default"

Reply:
The two issues have been resolved. Thank you!